caffe2/python/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ set(Caffe2_CPU_PYTHON_SRCS
 )
 
 # ---[ MKL binding
-if(USE_MKL AND USE_MKLML)
+if(USE_MKLML)
```

Comment:
You need to remove the match `endif()` too.

Reply:
So remove the if-question entirely?